artman/tasks/publish/local.py

```diff
@@ -73,7 +73,7 @@ def execute(self, gapic_code_dir, git_repo, local_paths,
         dests = []
 
         # Determine where the code belongs and stage it there.
-        for path in git_repo.get('paths', ['.']):
```

Comment:
Empty list is definitely not what we want, whereas `['.']` does make sense.
I understand what you are saying about the "opt-in" problem, though.

Maybe we should make it an error if `paths` is not set? (This proposal amounts to failing silently.)